crates/rmf_site_editor/src/site/physical_camera.rs

```diff
@@ -17,8 +17,10 @@
 
 use crate::site::*;
 use bevy::{core_pipeline::tonemapping::Tonemapping, prelude::*};
+use rmf_site_camera::{active_camera_maybe, ActiveCameraQuery};
 use rmf_site_format::{PhysicalCameraProperties, Pose};
-use rmf_site_picking::Selectable;
+use rmf_site_picking::{DoubleClickSelection, Select, Selectable, Selection};
+use std::time::Instant;
```

Comment:
Related to the above, `physical_camera.rs` has implementation of actual cameras in the site, not the "view camera", i.e. this:

<img width="2388" height="1380" alt="image" src="https://github.com/user-attachments/assets/3fafd44c-26f7-4f4b-a4d9-89cf888b53ca" />